### PR TITLE
Added testing from other branch

### PR DESCRIPTION
**Pull request by alexlwallis**

### CODE REVIEW

**Comment by alexlwallis** [comment]
> _No description provided._

>And UI additions.

File: client/src/index.jsx [hunk 1]
```diff
@@ -2,77 +2,139 @@ import React from 'react';
...
+      item2: []
     };
     this.veryFetching = this.veryFetching.bind(this);
+    this.switchUp = this.switchUp.bind(this);
   }
 
   componentWillMount(){
     let prodId = window.location.search.split("=")[1];
-    if (prodId){
+    if (prodId == 1){
```
Comment (Vaishnavi-Thiyagaraj) [inline client/src/index.jsx]: == is fine here but using === instead of == is a good coding practice.

File: client/src/index.jsx [hunk 2]
```diff
@@ -2,77 +2,139 @@ import React from 'react';
...
       console.log("prodId: ",prodId);
       fetch(`/suggested?prod_id=${prodId}`)
       .then(r => r.json())
       .then(d => {
+        let x = d.slice(0,5);
```
Comment (Vaishnavi-Thiyagaraj) [inline client/src/index.jsx]: No need to assign d.slice(0,5) in a variable if u r not reusing it. You can directly assign it to items1 in setState.

File: client/src/index.jsx [hunk 3]
```diff
@@ -2,77 +2,139 @@ import React from 'react';
...
 
   veryFetching(){
     fetch('/data')
     .then(r => r.json())
     .then(d => {
+      let x = d.slice(0,5);
+      let y = d.slice(5,10);
       this.setState({
-        items: d
+        item1: x,
```
Comment (Vaishnavi-Thiyagaraj) [inline client/src/index.jsx]: Same here

**Pull request merged**